squirrel/sqfuncproto.h

```diff
@@ -153,6 +153,7 @@ struct SQFunctionProto : public CHAINABLE_OBJ
     SQObjectPtr _name;
     SQUnsignedInteger   lang_features;
     SQInteger _stacksize;
+    SQInteger _hoistingLevel;
```

Comment:
Need to initialize this variable (in SQFunctionProto constructor or somewhere else) because in case of the old one-pass compiler it won't be initialized at all

Reply:
Done